app/_layout.tsx

```diff
@@ -1,11 +0,0 @@
-import { Stack } from 'expo-router'
-import React from 'react'
-import { StyleSheet, View } from 'react-native'
-
-export default function Layout() {
-  return (
-    <Stack>
-      <Stack.Screen name='index' options={{ headerShown: false }} />
-    </Stack>
-  )
-}
```

Comment:
why have you removed this as we would add header and any footer here in ?

Reply:
We do not require this footer to be in real design.
If any requirement for this top footer It will be resolved in the merge conflict.

Reply:
I added this file for now